integration-tests/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.fury.register-class-names=io.quarkiverse.fury.it.Struct
+quarkus.fury.register-classes."io.quarkiverse.fury.it.Foo".class-id=300
```

Comment:
Could you share en exmaple how to configure multiple class with class id and serializer in  `application.properties`?

Reply:
If we use `register_classes` instead of `register-classes`, will quarkus still parse config correctly?

Reply:
> Does it look like following code? If so, I perfer name configuration key as register-class instead of register-classes

Yeah, it is. I can change it to `register-class`.

> If we use register_classes instead of register-classes, will quarkus still parse config correctly?

No, it will not. I think Quarkus uses `microprofile` config spec.

Reply:
I see, let's change to `register-class`. And it would be great if we can add a serializer config here for test